content/flutter-publishing/publishing-to-app-store.md

```diff
@@ -69,7 +69,7 @@ Alternatively, if you wish to submit an already uploaded build for review in App
 3. To start the submission process, click **Prepare for Submission**.
 4. Check that your app metadata is up to date, and once everything is ready, click the **Submit for Review** button.
 
-Note that the application must be manually released to the App Store once Apple approves the release.
+Please note that when using the workflow editor, the application will be manually released to the App Store by default once it is approved by Apple. However, you have the option to set up automatic publishing by making the necessary changes in your `Apple developer account > Apps > Distribution settings`.
```

Comment:
Two things here:

1. Do you mean that apps will need to be released manually by users after Apple approves their submission
2. Where exactly is Apple developer account > Apps > Distribution settings` and what settings you are referring to ?

Reply:
@icarusdust can you please review it now, i have made the required changes to the path.

Reply:
I suggest you explain what release methods exist and then, where these methods can be configured in App Store Connect. By explaining it, you can highlight what the default behaviour is.

Reply:
@icarusdust, i have clarified what it means, please review